ikasaneip/developer/docs/DeveloperPreRequisites.md

```diff
@@ -100,7 +100,7 @@ This should reflect the Java JDK version you have just installed.
 
 For instance – the line in bold depicts the important information.
 
-Expected JDK 11 version output
+Expected JDK 8 version output
```

Comment:
Confused between @davidhilton68  @mitcje  changed.

3.1.x runs on JDK11 so why to change that to 8

Reply:
maybe we should remove all jdk 8 references for this document as its pinned to 3.1.x?